templates/include/address-controller.jsonnet

```diff
@@ -23,84 +43,208 @@ local common = import "common.jsonnet";
-              {
-                "image": image,
-                "name": "address-controller",
-                "env": [
-                  common.env("ENABLE_RBAC", enable_rbac),
-                  common.env("ENABLE_EVENT_LOGGER", enable_event_logger),
-                  common.env("ENVIRONMENT", environment),
-                  common.env("IMPERSONATE_USER", impersonate_user),
-                  common.env("ADDRESS_CONTROLLER_SA", address_controller_sa),
-                  common.env("ADDRESS_SPACE_ADMIN_SA", address_space_admin_sa),
-                  common.env("STANDARD_AUTHSERVICE_CONFIG", standard_authservice_config),
-                  common.env("WILDCARD_ENDPOINT_CERT_SECRET", wildcard_endpoint_cert_secret),
-                  common.env("RESYNC_INTERVAL", resync_interval),
-                  common.env("CHECK_INTERVAL", check_interval)
-                ],
-                "volumeMounts": [
-                  common.volume_mount("address-controller-cert", "/address-controller-cert", true),
-                ] + if template_config != "" then [ common.volume_mount("templates", "/enmasse-templates") ] else [],
-                "resources": {
-                    "requests": {
-                        "memory": "512Mi",
-                    },
-                    "limits": {
-                        "memory": "512Mi",
+        "spec": {
+          "serviceAccount": "enmasse-admin",
+          "containers": [
+            {
+              "image": images.address_controller,
+              "name": "address-controller",
+              "env": [
+                {
+                  "name": "ENABLE_RBAC",
+                  "valueFrom": {
+                    "configMapKeyRef": {
+                      "name": "address-controller-config",
+                      "key": "enableRbac",
+                      "optional": true
+                    }
+                  }
+                },
+                {
+                  "name": "ENABLE_EVENT_LOGGER",
+                  "valueFrom": {
+                    "configMapKeyRef": {
+                      "name": "address-controller-config",
+                      "key": "enableEventLogger",
+                      "optional": true
+                    }
+                  }
+                },
+                {
+                  "name": "ENVIRONMENT",
+                  "valueFrom": {
+                    "configMapKeyRef": {
+                      "name": "address-controller-config",
+                      "key": "environment",
+                      "optional": true
                     }
+                  }
                 },
-                "ports": [
-                  common.container_port("https", 8081),
-                  common.container_port("http", 8080)
-                ],
-               "livenessProbe": common.http_probe("https", "/apis/enmasse.io/v1/health", "HTTPS", 30),
-                "readinessProbe": common.http_probe("https", "/apis/enmasse.io/v1/health", "HTTPS", 30),
```

Comment:
@lulf The last param value (initialDelaySeconds: 30) is now not contained in the livenessProbe and readinessProbe definitions any more. Was that change intentional?

Reply:
Sort off. The plan was to see how it behaved, and I've not seen any issues with using the defaults yet, so I was thinking maybe to keep it.

Reply:
ok